mobile/src/main/res/values/strings.xml

```diff
@@ -481,4 +481,32 @@
     <string name="widget_type_video">Video</string>
     <string name="widget_type_mapview">Map</string>
     <string name="widget_type_chart">Chart</string>
+
+    <string name="tile">Tile</string>
+    <string name="tiles_for_quick_settings">Tiles for quick settings</string>
+    <string name="tile_empty">Empty tile</string>
+    <string name="tile_item_picker_title">Select an Item</string>
+    <string name="tile_configure_in_settings">This tile can be configured in the openHAB app settings</string>
+    <string name="tile_icon_openhab">openHAB icon</string>
+    <string name="tile_icon_tv">TV</string>
+    <string name="tile_icon_bulb">Bulb</string>
+    <string name="tile_icon_openhab_value" translatable="false">openhab_icon</string>
+    <string name="tile_icon_tv_value" translatable="false">tv</string>
+    <string name="tile_icon_bulb_value" translatable="false">bulb</string>
+    <string name="save">Save</string>
+    <string name="tile_number">Tile %d</string>
+    <string name="tile_error_saving">Please fill out every setting</string>
+    <string name="tile_disabled">Turned off</string>
+    <string name="tile_enable">Turn on</string>
+    <string name="item_item_and_action">Item and action</string>
+    <string name="tile_name">Name</string>
+    <string name="tile_icon">Icon</string>
+    <string name="tile_require_unlock">Require screen to be unlocked</string>
+    <string name="tile_require_unlock_summary_on">If your device is locked, you\'re prompted to unlock it, before the Item will be updated</string>
```

Comment:
`Prompt for unlocking the device before allowing Item updates`?

Reply:
Sounds better, picked